docs/user/traffic-analytics.rst

```diff
@@ -0,0 +1,47 @@
+Traffic Analytics
+=================
+
+Read the Docs aggregates statistics about visits to your documentation.
+This is mainly information about how often pages are viewed,
+and which return a `404 Not Found` error code.
+
+Traffic Analytics lets you see *which* documents your users are reading.
+This allows you to understand how your documentation is being used,
+so you can focus on expanding and updating parts people are reading most.
+
+To see a list of the top pages from the last month,
+go to the :guilabel:`Admin` tab of your project,
+and then click on :guilabel:`Traffic Analytics`.
+
+.. figure:: /_static/images/addons-analytics.png
+   :width: 50%
+   :align: center
+   :alt: Traffic analytics demo
+
+   Traffic analytics demo
+
+Enabling Traffic analytics
+--------------------------
+
+#. Go to the new dashboard:
+
+    * `Community <https://app.readthedocs.org>`_
+    * `Business <https://app.readthedocs.com>`_
+
```

Comment:
```suggestion
#. Go to the new :term:`dashboard`:
```

I like more that pattern that uses the glossary and its definition to link to both of the dashboards. When you hover on the `dashboard` word, it shows you both of the links:

![Peek 2024-10-02 12-51](https://github.com/user-attachments/assets/28197a1a-3181-40e2-a8df-1b7604043a96)

I've been following it in other places and I'm happy with it.

Reply:
Yea, that's a reasonable pattern until we build a better auto-redirect service 💯